src/fetch/transfer_file.py

```diff
@@ -13,6 +13,10 @@
 from src.utils.dataset import index_by
 
 
+def safe_url() -> str:
+    return f"https://gnosis-safe.io/app/eth:0xA03be496e67Ec29bC62F01a428683D7F9c204930/apps?appUrl=https://cloudflare-ipfs.com/ipfs/Qme49gESuwpSvwANmEqo34yfCkzyQehooJ5yL7aHmKJnpZ/"
```

Comment:
This can be done more cleanly over a few lines where you specify the safe address and app hash in their own lines and then format the url string with pre-exisiting variables.

Reply:
Also, the safe address should be of type `Address`.